src/test/unittest/Makefile

```diff
@@ -34,8 +34,8 @@
 # src/test/unittest/Makefile -- build unittest support library
 #
 TARGET = libut.a
-OBJS = ut.o ut_alloc.o ut_file.o ut_pthread.o ut_signal.o
-CFLAGS = -I../../include -ggdb -Wall -Werror -std=gnu99
+OBJS = ut.o ut_alloc.o ut_file.o ut_pthread.o ut_signal.o ../../common/util.o
```

Comment:
I think you should build the util.o in current directory as you may have different compiler flags. But since util.o is going away there is no problem.